windows/rngallery/AutolinkedNativeModules.g.cpp

```diff
@@ -1,5 +1,5 @@
 // AutolinkedNativeModules.g.cpp contents generated by "react-native autolink-windows"
-
+// clang-format off
```

Comment:
Why was this necessary?

Reply:
Not sure why that was added to the generated file. I'll remove it now.